Makefile

```diff
@@ -42,8 +42,12 @@ lint-cpp:
 	# clang-format --dry-run -Werror -i -style=file `find ./cpp/ -name "*.*pp"`
 	echo "C++ linting disabled for now"
 
+lint-docs:
+	mdformat docs/wiki/ README.md examples/README.md
```

Comment:
`lint-docs` should be:
- `python -m mdformat --check`
- `python -m codespell_lib <omit --write>`

Then seperately, add `fix-docs` and add to `fix` target.